packages/components/src/components/Search/Search.tsx

```diff
@@ -108,6 +115,7 @@ export const Search = ({ cancellable, onCancelPress, onChange, ...restProps }: S
 
 const themedStyles = EDSStyleSheet.create(theme => ({
     container: {
+        flex: 1,
```

Comment:
I would create a changeset for this fix too. it's possible to create multiple changesets per PR.

Also, it's nice if the changeset describes the bug you fixed. Looking at the current changeset, I can't really tell which issue you fixed related to onChange/onBlur